pom.xml

```diff
@@ -1448,15 +1449,27 @@
 
       <dependency>
         <groupId>org.apache.flink</groupId>
-        <artifactId>flink-clients_${scala.compat.version}</artifactId>
+        <artifactId>flink-clients</artifactId>
         <version>${flink.version}</version>
         <scope>test</scope>
       </dependency>
       <dependency>
         <groupId>org.apache.flink</groupId>
-        <artifactId>flink-streaming-java_${scala.compat.version}</artifactId>
+        <artifactId>flink-streaming-java</artifactId>
         <version>${flink.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.flink</groupId>
+        <artifactId>flink-java</artifactId>
+        <version>${flink.version}</version>
+        <exclusions>
+          <!-- excluding Kryo library due to Maven Enforcer convergence error -->
+          <exclusion>
+            <groupId>com.esotericsoftware.kryo</groupId>
```

Comment:
Can you take a look at which library pulls the same dependency and is causing conflict?
The best practice is to explicitly set the version for this library so that we won't run into issue (e.g. ClassNotFound) when only `flink` is included but not the other library that pulls this dependency.